src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -275,6 +279,21 @@ void FullyConnected::getSupportedDescriptors() {
 
     inDims = isDynamicNode() ? makeDummyInputDims() : getInputShapeAtPort(DATA_ID).getStaticDims();
     outDims = isDynamicNode() ? makeDummyOutputDims(inDims) : getOutputShapeAtPort(0).getStaticDims();
+#ifdef OV_CPU_WITH_MLAS
+    bool isINT8 = one_of(inputDataType, memory::data_type::u8, memory::data_type::s8) && weightsDataType == memory::data_type::s8;
+    // MLAS doesn't support post-ops fusing and BF16. INT8 is not enabled yet
+    // Disable MLAS when FC could fuse post
+    useMlas = !useSparseWeights && (inputDataType != memory::data_type::bf16) && !isINT8 && fusedWith.empty();
+    if (withBiases) {
+        const auto& biasDims = getInputShapeAtPort(BIAS_ID).getStaticDims();
+        bool isByChannel = biasDims.back() == outDims.back();
+        for (size_t i = 0; i < biasDims.size() - 1; i++) {
+            isByChannel = isByChannel && biasDims[i] == 1;
+        }
+        useMlas = useMlas && isByChannel;
+    }
+#endif
+    if (useMlas) return;
```

Comment:
Seems like we need to extend CPU debug capabilites to allow change gemm provider via env variable. Please create corresponding if not already.

Reply:
Should this debug capability be enabled only when `ifdef CPU_DEBUG_CAPS`, or this capability always exists and MLAS is selected by default.

Reply:
Yeah, I expect it should be under CPU_DEBUG_CAPS.
I don't insist to implement the feature in bounds if this PR. Follow-up ticket will be enough.